src/plugins/autoplay/README.md

```diff
@@ -0,0 +1,25 @@
+# Autoplay
+*This documentation here was not authored by the creator of the plugin*
```

Comment:
You can remove this. First of all this is reviewed by me and we are all valuable contributors here!